policies/booking_policy.rb

```diff
@@ -0,0 +1,25 @@
+class BookingPolicy
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-]+(\.[a-z]+)*\.[a-z]+\z/i
+
+  def initialize(params)
+    @user_email   = params[:user_email]
+    @first_class  = params[:first_class] || 0
```

Comment:
Just an FYI - an alternative looks like this: `params.fetch(:first_class, 0)` but it's used mostly during chaining e.g
`params.fetch(:key, {}).fetch(:another_key, {})...` to avoid errors in case there's no key present